README.md

```diff
@@ -193,6 +193,10 @@ For example: `client.list({'kind': 'user', 'inspect': True})`
 | search           | Search for resources based on specified query                |
 | inspect          | List the metadata for resources                              |
 
+### `check_privilege(kind, resource_id, privilege, role_id)`
+
+Checks for a privilege on a resource based on its kind, resource ID, and role ID. Returns a boolean.
+
```

Comment:
updated README.md

Reply:
Can we document that the role_id parameter is optional?